### ISSUE TITLE
__Secure cookie not removed after await HttpContext.SignOutAsync();

### ISSUE THREAD

**Issue by programmersommer** [aspnet/AspNetCore#17980]
### Describe the bug
I am using .NET Core 2.2

```
    services.AddAuthentication(options =>
                {
                    options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
                    options.DefaultChallengeScheme = CookieAuthenticationDefaults.AuthenticationScheme;
                })
                .AddCookie(options =>
                {
                    options.Cookie.HttpOnly = true;
                    options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
                    options.Cookie.IsEssential = true;
                    options.Cookie.Name = "__Secure-MyWebsite.Auth";
                    options.Cookie.SameSite = SameSiteMode.Lax;
                    options.LoginPath = "/Account/Index";
                    options.LogoutPath = null;
                    options.EventsType = typeof(CheckAuthTypeCookieAuthenticationEvents);
                })
```

When user is trying to log out with 
`    await HttpContext.SignOutAsync();`
he can't
It is also not possible to remove this cookie manually.
Everything works fine if remove "__" from Cookie name
`    options.Cookie.Name = "Secure-MyWebsite.Auth";`

**Comment by Tratcher** [comment]
I expect this is a duplicate of https://github.com/aspnet/AspNetCore/issues/17833 because the delete cookie doesn't get the `secure` attribute. You can confirm by capturing a Fiddler trace.

This has been fixed in 3.0 and we plan to patch in in 2.1. Note 2.2 is end-of-life and will likely not be patched. See https://github.com/aspnet/AspNetCore/issues/17833#issuecomment-565123537 for workarounds.

**Comment by programmersommer** [comment]
Thank you, @Tratcher 
Yes, logout works in 3.1
